extract/toc.es6

```diff
@@ -1,9 +1,10 @@
+import { join, dirname } from 'path-browserify';
 import items from './items';
 import getTocItem from './get-toc-item';
 import uniqueId from 'lodash/utility/uniqueId';
 
-const TAG = 'nav[id="toc"]';
-
+// Spec says we need to select the TOC using the epub:type=toc property
+const TAG = 'nav[epub\\\:type~=toc]';
```

Comment:
@dan-mckay we should test this in IE. There was a problem with the namespace. @neil-buckley was telling me that it might be because our books didn't include the epub namespace definitions. Anyway, worth testing it :)